src/card/color.rs

```diff
@@ -2,7 +2,7 @@
 use serde::{Deserialize, Serialize};
 
 /// Enum defining the 5 colors of magic
-#[derive(Serialize, Deserialize, Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug)]
```

Comment:
I think ordering colors might make sense. It's color wheel order here

Reply:
But is G less than or greater than W?